tests/typecheck/models/test_related_fields.yml

```
@@ -142,3 +142,51 @@
 
         class Other(models.Model):
             field = models.ForeignKey(MyModel, related_name="others", on_delete=models.CASCADE)
+
+
+- case: test_related_fields_with_two_generic_parameters
+  main: |
+        from myapp.models import Student
+        reveal_type(Student().school)  # N: Revealed type is "myapp.models.School"
+        reveal_type(Student().address)  # N: Revealed type is "myapp.models.Address"
```

Let's test the set type(`_ST`) argument results in no error(s) as well:

```suggestion
        from myapp.models import Address, School, Student
        reveal_type(Student().school)  # N: Revealed type is "myapp.models.School"
        reveal_type(Student().address)  # N: Revealed type is "myapp.models.Address"
        s = Student()
        s.school = School()
        s.address = Address()
```